src/test/java/bio/terra/service/filedata/DrsServiceTest.java

```diff
@@ -70,28 +92,46 @@ public void before() throws Exception {
             resourceService,
             configurationService,
             jobService,
-            performanceLogger);
+            performanceLogger,
+            profileService,
+            azureResourceConfiguration,
+            azureContainerPdao,
+            azureBlobStorePdao);
     when(jobService.getActivePodCount()).thenReturn(1);
     when(configurationService.getParameterValue(ConfigEnum.DRS_LOOKUP_MAX)).thenReturn(1);
 
     snapshotId = UUID.randomUUID();
     UUID fileId = UUID.randomUUID();
+    UUID bucketResourceId = UUID.randomUUID();
     DrsId drsId = new DrsId("", "v1", snapshotId.toString(), fileId.toString());
     drsObjectId = drsId.toDrsObjectId();
 
     SnapshotProject snapshotProject = new SnapshotProject();
     when(snapshotService.retrieveAvailableSnapshotProject(snapshotId)).thenReturn(snapshotProject);
 
-    fsFile =
+    azureFsFile =
         new FSFile()
             .createdDate(Instant.now())
             .description("description")
             .path("file.txt")
-            .gspath("gs://path/to/file.txt")
+            .cloudPath("https://core.windows.net/blahblah")
             .size(100L)
-            .fileId(fileId);
+            .fileId(fileId)
+            .bucketResourceId(bucketResourceId.toString());
     when(fileService.lookupSnapshotFSItem(snapshotProject, drsId.getFsObjectId(), 1))
-        .thenReturn(fsFile);
+        .thenReturn(azureFsFile);
```

Comment:
You can remove this line because you already have this mocked on line 178. Also, I think it would get overridden by line 133 anyways because it has the same inputs but it's returning the `googleFsFile`

Reply:
I'm going to make it mock a google and azure object separately